engine/access/rest/request/get_block.go

```diff
@@ -9,7 +9,7 @@ import (
 const heightQuery = "height"
 const startHeightQuery = "start_height"
 const endHeightQuery = "end_height"
-const MaxAllowedHeights = 50
+const MaxAllowedBlockRequestHeightRange = 50
```

Comment:
would the name `MaxRequestHeightRange` make sense (it's a bit more concise)?

Reply:
make sense - I changed the names to `MaxBlockRequestHeightRange` and `MaxEventRequestHeightRange` to distinguish between the two types of request.